backend/app/Repositories/EloquentSessionRepository.php

```diff
@@ -6,11 +6,35 @@
 
 use App\Repositories\Contracts\SessionRepository;
 use Illuminate\Support\Collection;
+use Illuminate\Support\Carbon;
+use App\Entities\Session;
+use App\Actions\Sessions\GetAvgSessionRequest;
+use App\Entities\Visitor;
 
 class EloquentSessionRepository implements SessionRepository
 {
     public function getCollection(): Collection
     {
         return collect([]);
     }
+
+    public function getAvgSession(GetAvgSessionRequest $request): int
+    {
+        $websiteId = auth()->user()->website->id;
+
+        $visitorsIDsOfWebsite = Visitor::where('website_id', $websiteId)
```

Comment:
This is a problem. Do query visitors from VisitorRepository in GetAvgSessionAction. And moreover, Marina adds website_id to Session entity, so it should be redone in the future. But for now move this to action